### PR TITLE
feat: update interview task

### PR DESCRIPTION
**Pull request by thomas-hs**
Fixes [Jira issue: HSP21-XXX](https://helloself.atlassian.net/browse/HSP21-XXX)

### Please label your commit using [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)

i.e.
fix: HSP21-123 Buttons displaying as lime green
feat: HSP21-123 Changed all buttons to hot pink

# Description: What was changed and why?

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

## Type of change

Please delete options that are not relevant.

-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.

-   [ ] Test A
-   [ ] Test B

# Front-end Checklist:

-   [ ] Are form validation errors correct?
-   [ ] Are API failure states handled?
-   [ ] Builds without warnings
-   [ ] Are there any console.log() in the code?
-   [ ] Does `yarn audit` flag any security issues?
-   [ ] Does the PR describe Acceptance Criteria? And does this ticket?
-   [ ] Appropriate tests?

# Back-end Checklist:

-   [ ] Is all response data tagged with appropriate permissions?
-   [ ] Do new endpoints make the correct permissions checks?
-   [ ] Does the PR describe Acceptance Criteria? And does this ticket?
-   [ ] New endpoints have been documented
-   [ ] Unit tests covering logic which could cause code to fail, especially when failure could go undetected.
-   [ ] Integration tests where appropriate

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,9 +4,9 @@ Thank you for taking the time to do this exercise. Please spend no longer than a
 
 Please fork this repository and send us your answer. Remember to provide us relevant permission to access the repository.
 
-## Brief: Trello Board Bookings
+## Brief: Kanban Board Bookings
 
-At HelloSelf, our operations team have to manage new bookings. We have a form to enter the member's details:
+At Hello, our operations team have to manage new bookings. We have a form to enter the member's details:
```
Comment (HelloSelfHannah) [inline README.md:9]:
```suggestion
An online mental health clinic is building a new board to allow its operations team to keep track of new bookings. They have a basic form to enter the member's details onto the board:
```
could be worth us being extra clear that this is theoretical here

File: README.md [hunk 2]
```diff
@@ -24,23 +24,24 @@ The board has 4 columns:
...
 
 ## Tasks
 
-1. Please add validations to the member form, and relevant error and helper messages.
+1. Please add validations to the member form, relevant error and helper messages.
 2. Once 'Submit' is pressed, a card will be created on under _Unclaimed_ column.
 3. The card will contain all the details entered into the form.
 4. Please save new card data to _local storage_.
 5. Previous entered cards will reload onto the board as saved in _local storage_.
 6. Please add styling to the form and page.
```
Comment (HelloSelfHannah) [inline README.md:39]: Might be worth taking page out here if we also mention it in bonus task 3

**Review by HelloSelfHannah — APPROVED** [review]
sweeet

File: README.md (hunk 2 above)
Comment (thomas-hs) [reply README.md:39]: I was thinking about that, but if they decide not to implement the bonus task then they might not see it

**Pull request merged**